README.md

```diff
@@ -255,3 +255,59 @@ The command used to shutdown the system.
 Set the environment variable `RIZZO_DEBUG` to `true`.
 
 example: `RIZZO_DEBUG=true vagrant status`
+
+# Commands
+
+## `rizzo generate`
+
+These command names are suggestions. Happy to entertain different ones.
+
+Will generate 'Vagrantfile' in your top control repo. Turn existing
+Vagrantfile into an erb.
+
+`spec/spec_helper.rb` will need at least the following.
```

Comment:
yeah, as part of the effort to allow for ensuring that we can compile catalogs for each role